bin/node/runtime/src/lib.rs

```diff
@@ -1473,6 +1475,67 @@ impl pallet_state_trie_migration::Config for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const AllianceMotionDuration: BlockNumber = 5 * DAYS;
+	pub const AllianceMaxProposals: u32 = 100;
+	pub const AllianceMaxMembers: u32 = 100;
+}
```

Comment:
We can do this kind of thing so that we don't need as many parameter_types: `type MaxConsumers = frame_support::traits::ConstU32<16>;` - the less magic macros the better.